pytorch_lightning/callbacks/progress/progress.py

```diff
@@ -206,6 +209,54 @@ def init_test_tqdm(self) -> tqdm:
         )
         return bar
 
+    def get_progress_bar_dict(
```

Comment:
this is incomplete. this only implements the `standard_metrics` here: what about `trainer.progress_bar_metrics` and merging them? https://github.com/PyTorchLightning/pytorch-lightning/blob/de57feff445744f431eaef29182b1bcbf58a1924/pytorch_lightning/trainer/properties.py#L302-L303